utils/measure/measure.py

```diff
@@ -912,23 +912,43 @@ def create(self) -> PowerMeter:
 def main():
     print(f"Powercalc measure: {_VERSION}\n")
 
+    device = inquirer.list_input("What kind of device do you want to measure the power of?",
+                              choices=['Light bulb(s)', 'Smart speaker', 'Other'])
+
     light_controller_factory = LightControllerFactory()
     power_meter_factory = PowerMeterFactory()
 
     try:
         power_meter = power_meter_factory.create()
         light_controller = light_controller_factory.create()
-    
+
         measure = Measure(light_controller, power_meter)
 
         args = sys.argv[1:]
-        if not args:
+
+        if device == 'Light bulb(s)':
             measure.start()
             exit(0)
-
-        if args[0] == "average":
+        elif device == 'Smart speaker':
+            try:
+                if args[0] == "average":
+                    duration = int(args[1])
+            except IndexError:
+                duration = 20
+            if inquirer.confirm('Ready to measure the standby-power? (Make sure your devices is in off or idle state in HA)'):
+                measure.measure_average(duration)
+            else:
+                exit(0)
+            print(f'Prepare to start measuring the power for {duration} seconds on each volume level starting with 10 until 100 (with steps of 10 between)')
+            print('Recommend to stream Pink Sound from https://www.genelec.com/audio-test-signals')
+            for volume in range(10,101, 10):
+                if inquirer.confirm(f'Set volume to {volume}% and confirm to start next {duration} second measurement'):
+                    measure.measure_average(duration)
```

Comment:
Would it be a nice last improvement for this first version to keep track of the volume level and recorded power measurements (in a dict) and than print a table in the end, so the user can easily add the values to model.json?